index.php

```diff
@@ -1,26 +1,22 @@
 <?php
 
-/**
- * Captcha for KIRBY3 Plugin
- *
- * @author Björn Fromme <mail@bjo3rn.com>
- * @author Markus Schatzl <marsch@mailbox.org>
- * @version 1.1.0
- */
-
```

Comment:
Why did you remove the docblock?

Reply:
This information is now contained in the mandatory composer.json, which seems the right place for it since Kirby3.